src/screens/General/About.js

```diff
@@ -1,76 +1,169 @@
 import React, { Component } from "react";
-import { View, SafeAreaView, FlatList, Dimensions } from "react-native";
+import { Avatar } from "react-native-elements";
 import { Actions } from "react-native-router-flux";
 import { ListItem } from "react-native-elements";
 import { NavBar } from "../../components/general";
 import Ionicons from "react-native-vector-icons/Ionicons";
+import { View, Text, SafeAreaView, FlatList, Dimensions, Linking, TouchableOpacity } from "react-native";
+import list from "../../data/AboutItems.json";
```

Comment:
Can you name this a more descriptive name that clearly defines what that list json is being used for inside of about